src/main/java/org/pac4j/vertx/handler/impl/CallbackHandler.java

```diff
@@ -51,75 +43,37 @@
     private final HttpActionAdapter httpActionHandler = new DefaultHttpActionAdapter();
     private final Vertx vertx;
     private final Config config;
+    private final boolean multiProfile;
 
     protected String defaultUrl = Pac4jConstants.DEFAULT_URL_VALUE;
```

Comment:
Yes, I can never remember that. In that case, I think it would be better to have several options classes:  `CallbackHandlerOptions`, `SecurityHandlerOptions` and `ApplicationLogoutHandlerOptions` as the options are really different from the three handlers.